src/servers/cron.ts

```diff
@@ -15,6 +15,7 @@ import { ColdStorage, Lightning, Wallets, Payments, Swap } from "@app"
 import { getCronConfig, TWO_MONTHS_IN_MS } from "@config"
 
 import { rebalancingInternalChannels, reconnectNodes } from "@services/lnd/utils-bos"
+import { extendSessions } from "@services/kratos"
```

Comment:
Servers should try to only access app layer

Reply:
fixed in 664a70632ec7c5cd00c54fafe559477bdf004c80